package-system/AWSGameLiftServerSDK/build_package_image.py

```diff
@@ -202,12 +192,7 @@ def configure_sdk_project(working_directory: WorkingDirectoryInfo,
 def build_sdk_project(source_folder: str,
                       build_folder: str,
                       build_type: str) -> None:
-    if PACKAGE_PLATFORM == PACKAGE_PLATFORM_OPTIONS[0]:
-        target: str = "--target ALL_BUILD"
-    elif PACKAGE_PLATFORM in (PACKAGE_PLATFORM_OPTIONS[1], PACKAGE_PLATFORM_OPTIONS[2]):
-        target: str = ""
-    else:
-        raise Exception(f"Error unsupported platform: {PACKAGE_PLATFORM}")
+    target: str = "--target aws-cpp-sdk-gamelift-server"
```

Comment:
nit: 
`target: str` may be overkill here since you are initializing it to a string. The type hints are useful for parameters usually.

Reply:
Thank you! Fixed